// svelte-scrolling/dist/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.scrollPosition = exports.scrollElement = exports.scrollBottom = exports.scrollTop = exports.setGlobalOptions = exports.scrollRef = exports.scrollTo = void 0;
var ScrollTo_1 = require("./actions/ScrollTo");
Object.defineProperty(exports, "scrollTo", { enumerable: true, get: function () { return __importDefault(ScrollTo_1).default; } });
var ScrollRef_1 = require("./actions/ScrollRef");
Object.defineProperty(exports, "scrollRef", { enumerable: true, get: function () { return __importDefault(ScrollRef_1).default; } });
var globalOptions_1 = require("./internal/globalOptions");
Object.defineProperty(exports, "setGlobalOptions", { enumerable: true, get: function () { return __importDefault(globalOptions_1).default; } });
var scrolling_1 = require("./functions/scrolling");
Object.defineProperty(exports, "scrollTop", { enumerable: true, get: function () { return scrolling_1.scrollTop; } });
Object.defineProperty(exports, "scrollBottom", { enumerable: true, get: function () { return scrolling_1.scrollBottom; } });
Object.defineProperty(exports, "scrollElement", { enumerable: true, get: function () { return scrolling_1.scrollElement; } });
Object.defineProperty(exports, "scrollPosition", { enumerable: true, get: function () { return scrolling_1.scrollPosition; } });
